Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -327,17 +327,42 @@ - (NSString *)formatProducts:(NSDictionary *)obj
 /// @name Video Tracking
 ///-------------------------
 
+
+/**
+ Event tracking for Adobe Heartbeats.
+
+ Must follow Segment's Video Spec to leverage Adobe Heartbeat tracking.
+ https://segment.com/docs/spec/video/
+
+ @param payload Payload sent on Segment `track` call
+ */
 - (void)trackHeartbeatEvents:(SEGTrackPayload *)payload
 {
     if ([payload.event isEqualToString:@"Video Playback Started"]) {
         self.config = [self createConfig:payload];
         self.ADBMediaHeartbeat = [self.ADBMediaHeartbeatFactory createWithDelegate:nil andConfig:self.config];
+        // createConfig can return nil if the Adobe required field
```

Comment:
createConfig confused me here - the method you are calling is createWithDelegate above. If it is indeed from the createConfig call even before then it feels like the check should be above the `createWithDelegate`

Reply:
I think you are commenting on an outdated commit. I caught that in self review and fixed [here](https://github.com/segment-integrations/analytics-ios-integration-adobe-analytics/pull/20/files#diff-174da67d34df04b252362301f24e0256R345)